tools/cythonize.py

```diff
@@ -100,6 +100,24 @@ def process_tempita_pyx(fromfile, tofile):
         f.write(pyxcontent)
     process_pyx(pyxfile, tofile)
 
+
+def process_tempita_pxi(fromfile, tofile):
+    try:
+        try:
+            from Cython import Tempita as tempita
```

Comment:
We should probably ignore the Cython version if they consider it an implementation detail. Should document `tempita` as a build dependency in `INSTALL.rst.txt`.